plugins/chain_api_plugin/chain.swagger.yaml

```diff
@@ -89,6 +89,171 @@ paths:
             application/json:
               schema:
                 $ref: "https://eosio.github.io/schemata/v2.1/oas/BlockInfo.yaml"
+  /get_finalizer_info_result:
+    post:
+      description: Returns information about finalizers
+      operationId: get_finalizer_info_result
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              type: object
+              nullable: true
+      responses:
+        "200":
+          description: OK
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  active_finalizer_policy:
+                    type: object
+                    description: List of current finalizers
+                    properties:
+                      generation:
+                        type: integer
+                        description: version number of finalizers set
```

Comment:
generation number of finalizer policy. Activated finalizer policy generation numbers are always increasing, but may have gaps.